> Solving the next problem at: 2024-09-27 08:34:29 UTC
> Search algorithm: bfs
> Instance ID=blocks/pprobBLOCKS-4-0-err-rate-0-5
> Plan length=6
> Domain class: blocks
> Instance name: pprobBLOCKS-4-0-err-rate-0-5
> Ground plan:
['(pick-up b)', '(stack b a)', '(pick-up c)', '(stack c b)', '(pick-up d)', '(stack d c)']
> Lifted plan:
['pick-up', 'stack', 'pick-up', 'stack', 'pick-up', 'stack']
>>>  Vanilla ground repair length:
0

>>>  Vanilla ground repair:


An error occurred during A* search: name 'logger' is not defined
Traceback (most recent call last):
  File "/home/nader/projects/lifted-white-plan-domain-repair/instance_solver.py", line 70, in solve_instance
    path, goal_node = searcher.find_path(log_interval=log_interval)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nader/projects/lifted-white-plan-domain-repair/search_partial_grounding/search_algorithms.py", line 49, in find_path
    for neighbor in current_node.get_neighbors():
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nader/projects/lifted-white-plan-domain-repair/search_partial_grounding/node.py", line 167, in get_neighbors
    logger.debug(f'> Current state:\n{self.current_state}')
    ^^^^^^
NameError: name 'logger' is not defined. Did you mean: 'self.logger'?

> Time spent on this problem: 0.0 hours = 0.0 minutes = 0.20 seconds
> Subprocess finished.
